domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/vilkår/kvp/KVPVilkår.kt

```diff
@@ -0,0 +1,82 @@
+package no.nav.tiltakspenger.saksbehandling.domene.vilkår.kvp
+
+import no.nav.tiltakspenger.libs.periodisering.Periode
+import no.nav.tiltakspenger.libs.periodisering.PeriodeMedVerdi
+import no.nav.tiltakspenger.libs.periodisering.Periodisering
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.Lovreferanse
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.SkalErstatteVilkår
+import no.nav.tiltakspenger.saksbehandling.domene.vilkår.Utfall
+
+/**
+ * Kvalifiseringsprogrammet (KVP): https://www.nav.no/kvalifiseringsprogrammet
+ *
+ * @param søknadSaksopplysning Saksopplysninger som kan være avgjørende for vurderingen. Kan ikke ha hull. [avklartSaksopplysning]/faktumet er den avgjørende saksopplysningen.
+ * @param avklartSaksopplysning Faktumet som avgjør om vilkåret er oppfylt eller ikke. Null implisiserer uavklart.
+ * @param utfall Selvom om utfallet er
+ *
+ */
+data class KVPVilkår private constructor(
+    val søknadSaksopplysning: KvpSaksopplysning,
+    val saksbehandlerSaksopplysning: KvpSaksopplysning?,
+    val avklartSaksopplysning: KvpSaksopplysning,
+    val utfall: Periodisering<Utfall>?,
```

Comment:
Da Erlend og Karl-Evald lagde vurderinger for tiltak, så brukte de detaljer-feltet aktivt. Du har bare Periodisering<Utfall>, som sånn sett er litt mer begrenset. Jeg ville helst bare ha Periodisering<Utfall> jeg også, men jeg anbefaler at vi iallefall vurderer usecaset på tiltak..

Reply:
Når vi kommer til tiltaksdeltagelse, så høres det ut som vi får en egen type ÅrsakMedUtfall eller lignende. Eller at vi legger de ved siden av hverandre. Det må vi se på når vi kommer dit. Det er ikke sikkert KvpVilkår trenger gjenspeile dette.